check_pgactivity

```diff
@@ -5551,6 +5558,18 @@ sub check_streaming_delta {
 
     @rs = @{ query_ver( $hosts[0], %queries ) };
 
+    my $index = 0;
```

Comment:
I see. Maybe add a new value like "#of temp slave" with only matched value from parameters could be a solution ?